src/test/java/uk/gov/hmcts/reform/slc/health/PdfServiceHealthIndicatorTest.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.slc.health;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.boot.actuate.health.Status;
+import uk.gov.hmcts.reform.pdf.service.client.PDFServiceClient;
+import uk.gov.hmcts.reform.pdf.service.client.exception.PDFServiceClientException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PdfServiceHealthIndicatorTest {
+
+    @Mock
+    private PDFServiceClient client;
+
+    private HealthIndicator healthIndicator;
+
+    @Before
+    public void setUp() {
+        healthIndicator = new PdfServiceHealthIndicator(client);
+    }
+
+    @Test
+    public void pdf_service_should_be_healthy() {
+        given(client.serviceHealthy()).willReturn(Health.up().build());
+
+        assertThat(healthIndicator.health().getStatus()).isEqualTo(Status.UP);
```

Comment:
How about `should_return_healthy_status_when_client_returns_healthy_status`? Or something nicer, but mentioning the conditions. Same for the unhealthy status.